classes/search/ArticleSearchDAO.php

```diff
@@ -92,7 +92,7 @@ public function getPhraseResults($journal, $phrase, $publishedFrom = null, $publ
             FROM
                 submissions s
                 JOIN publications p ON (p.publication_id = s.current_publication_id)
-                JOIN publication_settings ps ON (ps.publication_id = p.publication_id AND ps.setting_name=\'issueId\' AND ps.locale=\'\')
+                JOIN issues i ON (i.issue_id = p.issue_id AND i.journal_id = s.context_id)
```

Comment:
You forgot to remove the line below. Also, I think we can trust enough in the code to drop the `AND i.journal_id = s.context_id`